Removed blas output

bwingconda opened the pull request:
Fixes an issue publishing with the new PBP.

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_a03aoeklx5/clone` failed. (See above for error)
The following problems have been found:

===== WARNINGS =====
- clone/recipe/meta.yaml:1: missing_description: The recipe is missing a description
- clone/recipe/meta.yaml:1: missing_license_file: The recipe is missing the `about/license_file` or `about/license_url` key.

===== ERRORS =====
- clone/recipe/meta.yaml:1: missing_tests: output "openblas": No tests were found.
- clone/recipe/meta.yaml:28: patch_unnecessary: patch should not be in build when source/patches is not set
- clone/recipe/meta.yaml:0: missing_section: The about section is missing.
- clone/recipe/meta.yaml:1: missing_tests: output "openblas-devel": No tests were found.
- clone/recipe/meta.yaml:0: missing_section: output "libopenblas-static": The requirements section is missing.
- clone/recipe/meta.yaml:1: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:1: missing_tests: output "nomkl": No tests were found.
- clone/recipe/meta.yaml:1: missing_summary: The recipe is missing a summary
- clone/recipe/meta.yaml:1: missing_documentation: The recipe is missing a doc_url or doc_source_url
- clone/recipe/meta.yaml:1: missing_home: The recipe is missing a homepage URL
- clone/recipe/meta.yaml:1: missing_license: The recipe is missing the `about/license` key.
- clone/recipe/meta.yaml:1: missing_dev_url: The recipe is missing a dev_url
===== Final Report: =====
12 Errors and 2 Warnings were found

</details>

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_27pc6kpggt/clone` failed. (See above for error)
The following problems have been found:

===== WARNINGS =====
- clone/recipe/meta.yaml:1: missing_description: The recipe is missing a description
- clone/recipe/meta.yaml:1: missing_license_file: The recipe is missing the `about/license_file` or `about/license_url` key.

===== ERRORS =====
- clone/recipe/meta.yaml:1: missing_dev_url: The recipe is missing a dev_url
- clone/recipe/meta.yaml:28: patch_unnecessary: patch should not be in build when source/patches is not set
- clone/recipe/meta.yaml:1: missing_summary: The recipe is missing a summary
- clone/recipe/meta.yaml:1: missing_tests: output "nomkl": No tests were found.
- clone/recipe/meta.yaml:1: missing_tests: output "openblas": No tests were found.
- clone/recipe/meta.yaml:1: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:1: missing_documentation: The recipe is missing a doc_url or doc_source_url
- clone/recipe/meta.yaml:1: missing_home: The recipe is missing a homepage URL
- clone/recipe/meta.yaml:0: missing_section: The about section is missing.
- clone/recipe/meta.yaml:1: missing_license: The recipe is missing the `about/license` key.
- clone/recipe/meta.yaml:0: missing_section: output "libopenblas-static": The requirements section is missing.
- clone/recipe/meta.yaml:125: version_constraints_missing_whitespace: output "openblas": Packages and their version constraints must be space separated
- clone/recipe/meta.yaml:1: missing_tests: output "openblas-devel": No tests were found.
===== Final Report: =====
13 Errors and 2 Warnings were found

</details>

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_31bepdhoza/clone` failed. (See above for error)
The following problems have been found:

===== WARNINGS =====
- clone/recipe/meta.yaml:1: missing_license_file: The recipe is missing the `about/license_file` or `about/license_url` key.
- clone/recipe/meta.yaml:1: missing_description: The recipe is missing a description

===== ERRORS =====
- clone/recipe/meta.yaml:125: version_constraints_missing_whitespace: output "openblas": Packages and their version constraints must be space separated
- clone/recipe/meta.yaml:1: missing_summary: The recipe is missing a summary
- clone/recipe/meta.yaml:0: missing_section: output "libopenblas-static": The requirements section is missing.
- clone/recipe/meta.yaml:1: missing_home: The recipe is missing a homepage URL
- clone/recipe/meta.yaml:0: missing_section: The about section is missing.
- clone/recipe/meta.yaml:1: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:28: patch_unnecessary: patch should not be in build when source/patches is not set
- clone/recipe/meta.yaml:1: missing_license: The recipe is missing the `about/license` key.
- clone/recipe/meta.yaml:1: missing_dev_url: The recipe is missing a dev_url
- clone/recipe/meta.yaml:1: missing_tests: output "nomkl": No tests were found.
- clone/recipe/meta.yaml:1: missing_documentation: The recipe is missing a doc_url or doc_source_url
- clone/recipe/meta.yaml:1: missing_tests: output "openblas-devel": No tests were found.
- clone/recipe/meta.yaml:1: missing_tests: output "openblas": No tests were found.
===== Final Report: =====
13 Errors and 2 Warnings were found

</details>

bwingconda committed:
Removed blas output
Remove subpackage pinning in favor of explicit pinning
Ignore overdepending errors

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_e2uvrovdey/clone` failed. (See above for error)
The following problems have been found:

===== WARNINGS =====
- clone/recipe/meta.yaml:1: missing_license_file: The recipe is missing the `about/license_file` or `about/license_url` key.
- clone/recipe/meta.yaml:1: missing_description: The recipe is missing a description

===== ERRORS =====
- clone/recipe/meta.yaml:1: missing_tests: output "openblas": No tests were found.
- clone/recipe/meta.yaml:1: missing_documentation: The recipe is missing a doc_url or doc_source_url
- clone/recipe/meta.yaml:1: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:1: missing_summary: The recipe is missing a summary
- clone/recipe/meta.yaml:1: missing_dev_url: The recipe is missing a dev_url
- clone/recipe/meta.yaml:125: version_constraints_missing_whitespace: output "openblas": Packages and their version constraints must be space separated
- clone/recipe/meta.yaml:1: missing_tests: output "openblas-devel": No tests were found.
- clone/recipe/meta.yaml:28: patch_unnecessary: patch should not be in build when source/patches is not set
- clone/recipe/meta.yaml:1: missing_home: The recipe is missing a homepage URL
- clone/recipe/meta.yaml:0: missing_section: output "libopenblas-static": The requirements section is missing.
- clone/recipe/meta.yaml:1: missing_tests: output "nomkl": No tests were found.
- clone/recipe/meta.yaml:1: missing_license: The recipe is missing the `about/license` key.
- clone/recipe/meta.yaml:0: missing_section: The about section is missing.
===== Final Report: =====
13 Errors and 2 Warnings were found

</details>

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_d9_11qhtnc/clone` failed. (See above for error)
The following problems have been found:

===== WARNINGS =====
- clone/recipe/meta.yaml:1: missing_license_file: The recipe is missing the `about/license_file` or `about/license_url` key.
- clone/recipe/meta.yaml:1: missing_description: The recipe is missing a description

===== ERRORS =====
- clone/recipe/meta.yaml:1: missing_license: The recipe is missing the `about/license` key.
- clone/recipe/meta.yaml:28: patch_unnecessary: patch should not be in build when source/patches is not set
- clone/recipe/meta.yaml:1: missing_tests: output "openblas-devel": No tests were found.
- clone/recipe/meta.yaml:0: missing_section: The about section is missing.
- clone/recipe/meta.yaml:1: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:1: missing_summary: The recipe is missing a summary
- clone/recipe/meta.yaml:1: missing_dev_url: The recipe is missing a dev_url
- clone/recipe/meta.yaml:0: missing_section: output "libopenblas-static": The requirements section is missing.
- clone/recipe/meta.yaml:125: version_constraints_missing_whitespace: output "openblas": Packages and their version constraints must be space separated
- clone/recipe/meta.yaml:1: missing_tests: output "openblas": No tests were found.
- clone/recipe/meta.yaml:1: missing_tests: output "nomkl": No tests were found.
- clone/recipe/meta.yaml:1: missing_documentation: The recipe is missing a doc_url or doc_source_url
- clone/recipe/meta.yaml:1: missing_home: The recipe is missing a homepage URL
===== Final Report: =====
13 Errors and 2 Warnings were found

</details>
